README.md

```diff
@@ -1,2 +1,85 @@
-# docker-hub-php-cli
-Different PHP versions as CLI for CI
+# PHP CLI
+[![CircleCI](https://circleci.com/gh/final-gene/docker-hub-php-cli/tree/master.svg?style=svg)](https://circleci.com/gh/final-gene/docker-hub-php-cli/tree/master) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e067ba9a720d4b3995c21adc9182f599)](https://www.codacy.com/app/final-gene/docker-hub-php-cli?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=final-gene/docker-hub-php-cli&amp;utm_campaign=Badge_Grade)
+
+This is a image to run the [PHP CLI](http://php.net/manual/en/features.commandline.php).
+
+## Supported tags and respective Dockerfile links
+* `7.2`, `7.2-xdebug`, `latest` [(7.2/Dockerfile)](https://github.com/finalgene/docker-hub-php-cli/blob/master/7.2/Dockerfile)
```

Comment:
❓ Why the additional tag for xdebug?

Reply:
This is legacy to avoid breaking existing usages of my images.